src/server/routes/api/classrooms/index.js

```diff
@@ -255,25 +286,39 @@ router.post("/:classroomId/ask", async (req, res) => {
     return res.json({ success: false, message: "You are not a member of that classroom" });
   }
 
+  const updatedClassroom = await Classrooms.findById(req.params.classroomId);
+  const usersQuestions = updatedClassroom.questions.filter( q => 
+    (q.user.toString() === req.user.id.toString()) 
+    && (q.question === question)
+  );
+  const newQuestion = usersQuestions[usersQuestions.length-1];
+
+  req.app.ee.emit("new_question", {
```

Comment:
Maybe emit this event with no data. the client can hit the rest API to get the most recently updated list of questions and answers. That way it is guaranteed to be up to date even if the events are delivered out of order

Reply:
Now that i’m thinking about this i’m not sure i want to force the client to pull the rest api every time. We can just merge this then talk about it later. We should be aware this is a potential problem point though

Reply:
See my response for https://github.com/csi4999-panic-button/panic-button/pull/48#discussion_r151499075